{
  "Header": {
    "Original": "CVE-2024-12254 (Starting in Python 3.12.0, the asyncio._SelectorSocketTransport.writel ...)",
    "ID": "CVE-2024-12254",
    "Description": "(Starting in Python 3.12.0, the asyncio._SelectorSocketTransport.writel ...)"
  },
  "Annotations": [
    {
      "Original": "- python3.13 \u003cunfixed\u003e",
      "Type": "package",
      "Package": "python3.13",
      "Kind": "unfixed"
    },
    {
      "Original": "- python3.12 \u003cunfixed\u003e",
      "Type": "package",
      "Package": "python3.12",
      "Kind": "unfixed"
    },
    {
      "Original": "- python3.11 \u003cnot-affected\u003e (Vulnerable code introduced in 3.12)",
      "Type": "package",
      "Package": "python3.11",
      "Kind": "not-affected",
      "Description": "Vulnerable code introduced in 3.12"
    },
    {
      "Original": "- python3.9 \u003cnot-affected\u003e (Vulnerable code introduced in 3.12)",
      "Type": "package",
      "Package": "python3.9",
      "Kind": "not-affected",
      "Description": "Vulnerable code introduced in 3.12"
    },
    {
      "Original": "NOTE: https://github.com/python/cpython/issues/127655",
      "Type": "NOTE",
      "Description": "https://github.com/python/cpython/issues/127655"
    },
    {
      "Original": "NOTE: https://github.com/python/cpython/pull/127656",
      "Type": "NOTE",
      "Description": "https://github.com/python/cpython/pull/127656"
    },
    {
      "Original": "NOTE: https://mail.python.org/archives/list/security-announce@python.org/thread/H4O3UBAOAQQXGT4RE3E4XQYR5XLROORB/",
      "Type": "NOTE",
      "Description": "https://mail.python.org/archives/list/security-announce@python.org/thread/H4O3UBAOAQQXGT4RE3E4XQYR5XLROORB/"
    },
    {
      "Original": "NOTE: https://github.com/python/cpython/commit/71e8429ac8e2adc10084ab5ec29a62f4b6671a82 (3.13-branch)",
      "Type": "NOTE",
      "Description": "https://github.com/python/cpython/commit/71e8429ac8e2adc10084ab5ec29a62f4b6671a82 (3.13-branch)"
    },
    {
      "Original": "NOTE: https://github.com/python/cpython/commit/9aa0deb2eef2655a1029ba228527b152353135b5 (3.12-branch)",
      "Type": "NOTE",
      "Description": "https://github.com/python/cpython/commit/9aa0deb2eef2655a1029ba228527b152353135b5 (3.12-branch)"
    },
    {
      "Original": "TODO: Double-check affected status for python3.11",
      "Type": "TODO",
      "Description": "Double-check affected status for python3.11"
    }
  ]
}